### ISSUE TITLE
Watch Your ES6: Old Node Requires Old-Style Code

### ISSUE THREAD

**Issue by NAnguiano** [dondi/GRNsight#410]
The version of node being run on the server is extremely old: 0.10. @dondi is going to upgrade it eventually (issue #407), but for now, there's many things that simply just don't work when they're pushed to the server. Most notably, `for of` (ex: `for (var potato of potatoArray) {`), will not work on the current version. This would not show up in testing, and @dondi and I were not aware of it until we tried to refresh beta on 2/27. If you're using `for of`, it's safest to remove it for now or replace it with a normal for loop, as it will cause the server to crash when a file is run.

Basically, for now, until @dondi can update our server, code using plain ol' Javascript, with none of the exciting stuff.

**Comment by kdahlquist** [comment]
I'm setting up check boxes for everyone to acknowledge this notification so that we can close this issue on Thursday.

- [ ] @mihirsamdarshi 
- [ ] @anuvarsh 
- [ ] @ebachoura 
- [ ] @yshin4 
- [ ] @eileenchoe 
- [ ] @dondi

**Comment by kdahlquist** [comment]
Got verbal acknowledgment about this at Thursday's meeting.
